docs/spl-docs/evaluation-functions/mathematical-functions.md

```diff
@@ -0,0 +1,441 @@
+
+This allows the user to categorize transactions efficiently, aiding in risk assessment and detailed financial reporting.
+
+## pow(\<num>, \<exp>)
+
+This function returns a number to the power of the exponent.
+
+### Usage
+
+The `<num>` argument can be the name of a numeric field or a numeric literal.
+
+The `<exp>` argument is the exponent.
+
+### Example
+
+The following example calculates the area of a circle, which is `pi()` multiplied by the radius to the power of 2.
+
+```
+... | eval area_circle=pi()*pow(radius,2)
+```
+
+### Use-Case Example
+
+**Calculate future investment value with compound interest**
+
+**Problem:** A user wants to calculate the future value of an investment based on compound interest. The formula for compound interest is given by `Future_Value = Principal * (1 + annual_interest_rate/times_compounded_per_year)^(times_compounded_per_year * years)`.
+
+**Solution:** The `pow()` command can be used to compute `(1 + annual_interest_rate/times_compounded_per_year)^(times_compounded_per_year * years)`.
+
+```
+index=investment_data
+| eval principal = 1000
+| eval annual_interest_rate = 0.05
+| eval times_compounded_per_year = 4
+| eval years = 10
+| eval future_value = principal * pow(1 + annual_interest_rate / times_compounded_per_year, times_compounded_per_year * years)
+| fields principal, annual_interest_rate, times_compounded_per_year, years, future_value
+```
+
+## pi()
+
+This function takes no arguments and returns the constant `pi` to 11 digits of precision.
+
+### Usage
+
+This function is useful whenever the precise value of π is required, eliminating the need to manually enter its value.
+
+### Example
+
+The following example rounds up the Pi value to two decimal places and assigns it to a new field called `pi_rounded`.
+
+```
+... | eval pi_rounded = round(pi(), 2)
+```
+
+### Use-Case Example
+
+**Calculate the area of a circle**
+
+**Problem:** A user wants to calculate the area of a circular field given the radius. The formula for the area of a circle is `Area = pi() * radius^2`.
+
+**Solution:** The `pi()` command can be used to get the precise value of π for the area calculation.
+
+```
+index=circle_data
+| eval radius = 5
+| eval area_circle = pi() * pow(radius, 2)
+| fields radius, area_circle
+```
+
+**Explanation:**
+1. The `eval` command assigns the radius value.
+2. The `eval` command calculates the area of the circle using `pi() * pow(radius, 2)`.
+
+This ensures that the user can accurately calculate the area of a circle given its radius using the precise value of π.
+
+## sqrt(\<num>)
+
+This function returns the square root of a number.
+
+### Usage
+
+The `<num>` argument can be the name of a numeric field or a numeric literal.
+
+### Example
+
+The following example returns `3`:
+
+```
+... | eval n=sqrt(9)
+```
+
+### Use-Case Example
+
+**Calculate the hypotenuse of a right triangle**
```

Comment:
nit: maybe change this to something a bit more practical, like calculating standard deviation of something (number of requests, latency, etc.)

Reply:
Calculating standard deviation requires using lot of `multi value` commands like `mvsum`, `mvmap`, `mvcount`, etc. Which I do not think we support yet. And also the command `stats values(latency)`, I do not think the output is correct.  

So for now I have used something else called `Root Mean Square Error (RMSE) of predictions`.